### DISCUSSION TITLE
Split day reimbursements

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#32] [Q&A] [answered]
Rather than splitting each variation of split day payments into their own line on the report, would it be acceptable that we maintain one line on the report and list the split reimbursement terms in the algorithm field, followed by the estimated allowed amounts for each negotiated payer?  

I am working on behalf of several health systems who utilize our contracting product and trying to programmatically create the new file based on their current contracts. The amount of split days variation from payer to payer within a single hospital, for the same service is quite large. Pulled a random example, and for this one facility just for a c-section, there would need to be 11 lines total on the csv wide format. The vaginal delivery service would need to break into 9 lines and then for their variety of transplants, 35 lines if each variation had to have their own line.  Some of those lines would only apply to one negotiated payer rate.  There are also examples where the negotiated payer rate is the lesser of: a 3 day flat rate and 4th+ per diem Or the workers comp reimbursement, so it makes deciding what to do with that programmatically, more complicated.  

In my opinion, these reimbursements fall into the definition for algorithm in the sense that not all patients will have the same length of stay for that one service category, and by reporting the full service reimbursement in the algorithm field and providing the estimated allowed amount for each negotiated payor contract, keeps the logic more consistent across the board and it is easier to compare rates in their totality across payers for the same service. 

Please advise if that would be acceptable.

**Comment by RidenourK** [comment]
@JaSimps5. Thank you for your question. 

Please refer to the data encoded under the “Description” header in rows 9 through 11 of the “[CSV Wide Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/CSV/Wide%20Format%20Examples)” on the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency) for an example of how to encode data in this scenario.

**Reply by JaSimps5** [reply]
@RidenourK I see the examples, my question is can we have an alternate way to do this?  The example is pretty clean but that doesn't necessarily reflect the reality of contracting examples with this reimbursement methodology. 
I have attached an example of a CSECT procedure with the different split reimbursements across a handful of Negotiated rates for one hospital. In my attachment, I have it split like in the github example as well as what I am suggesting.  The main difference is having one row vs multiple and listing the estimated amount that would provide better insight as to what the payers reimburse overall for the CSECT procedure at this facility.  

I didn't add to my example the situation I listed in my initial question where the reimbursement is the lesser of: a 3 day flat rate and 4th+ per diem Or the workers comp reimbursement.  That lesser of is based on the entire claim and not the first 3 days or the 4+ days, so I'm not even sure, other than putting this in an algorithm, what the expectations are there. 

ps.I only populated the fields that were relevant to my example reimbursement 

[Split Day example.csv](https://github.com/CMSgov/hospital-price-transparency/files/14128462/Split.Day.example.csv)

**Reply by RidenourK** [reply, marked as answer]
@JaSimps5. Thank you for your question. 

CMS is unable to comment on or provide individualized/case-by-case predeterminations as to whether a particular strategy or approach would be deemed compliant. The compliance process typically involves a comprehensive compliance review in response to a CMS audit or a complaint received through the Hospital Price Transparency website.
